AnkiConnect.py

```diff
@@ -434,60 +434,60 @@ def guiAddCards(self):
         addCards.activateWindow()
 
 
-    def getNextCard(self):
-        if (self.window().reviewer.card is None) or (self.window().state != "review"):
+    def guiGetNextCard(self):
+        if self.window().reviewer.card is None or self.window().state != 'review':
             self.window().moveToState('review')
 
         card = self.window().reviewer.card
-        nextCard = {}
-        nextCard['id'] = card.id
-        nextCard['question'] = card._getQA()['q']
-        nextCard['answer'] = card._getQA()['a']
-        nextCard['answerButtons'] = self.window().reviewer._answerButtonList()
-        nextCard['model_name'] = card.note(reload)._model['name']
-        nextCard['ord'] = card.ord
-        nextCard['fields'] = json.dumps(card.note(reload).fields)
-        nextCard['_fmap'] = card.note(reload)._fmap
-        nextCard['success'] = 'true'
-        return json.dumps(nextCard)
-
-
-    def showQuestion(self):
-        if (self.window().reviewer.card is None) or (self.window().state != "review"):
+        return {
+            'success': True,
+            'id': card.id,
+            'question': card._getQA()['q'],
+            'answer': card._getQA()['a'],
+            'answerButtons': self.window().reviewer._answerButtonList(),
+            'modelName': card.note(reload)._model['name'],
+            'ord': card.ord,
```

Comment:
Still not sure what `card.ord` is. If it's something useful, it should be renamed as such, otherwise it should not be exposed in the API.